.github/workflows/vm.yml

```diff
@@ -137,6 +150,18 @@ jobs:
                 ;;
             esac
 
+      - name: Fix git ownership
+        uses: cross-platform-actions/action@v0.30.0
+        with:
+          operating_system: ${{ matrix.os_image }}
+          version: ${{ matrix.os_version }}
+          environment_variables: ASAN_OPTIONS AUTOCONF_VERSION AUTOMAKE_VERSION
+          shutdown_vm: false
+          sync_files: true
+          run: |
+            # Workaround https://github.com/cross-platform-actions/action/issues/108
```

Comment:
Nit: either `Work around` (with a space) as a verb or `Workaround for` as a noun.